### PR TITLE
SMB v 22.8 - Add-on for Sao Mai Braille program

### PR DESCRIPTION
**Pull request by manhcuong0312**
### Release information
- Name (Example, Add-on Updater): SMB
- Author (example, Your Name): Sao Mai Technology Team
- Repo (example, https://github.com/yourname/addonName): https://github.com/Sao-Mai-Center/SMB
- Version (example, 21.07): 22.8
- Update channel (example, stable, dev or stable and dev): stable
- NVDA compatibility (example, 2021.1 and beyond): 2019.3 and beyond):

### Changelog (mention changes in separate lines starting with dash space)
- 2022.8: compatible with NVDA 2022.3
### Additional information
Sao Mai Braille (SMB) is a free rich text editing and Braille translation software for Windows, developed by [Sao Mai Center for the Blind](https://www.saomaicenter.org/en)
This is a small add-on to make NVDA screen reader more accessible to SMB's interface.
More info about this program can be found at <https://www.saomaicenter.org/en/smsoft/smb>

### CODE REVIEW

**Review by josephsl — CHANGES_REQUESTED** [review]
Hi,

A few things:

* Have you contacted the app developer to see if they can improve things without use of add-ons?
* Note that smb add-on comes before other add-ons in add-on keys map. I advise moving "smb" entry to somewhere else in the add-ons array so it can be sorted in alphabetical order if possible (perhaps between Sentence Nav and Sound Sound Splitter).
* For future reference, for pull requests, please please the name of the add-on and its version in the pull request title as "update get.php" may not be helpful for folks (may have come from the commit message, which I understand; I think we might as well have a discussion on NVDA add-ons list about descriptive commit messages).

Hope this helps. Thanks.

**Comment by manhcuong0312** [comment]
- Have you contacted the app developer to see if they can improve things without use of add-ons?
  Sao Mai Center for the blind is my office. Right now the program is not accessible to NVDA in its main interface, and we've not find solution for it yet so we write that add-on, and my boss suggest m e to add it in NVDA add-on's community so people can get it more easier.
- I advise moving "smb" entry to somewhere else in the add-ons array...
- that smb add-on comes before other add-ons in add-on keys map. I advise moving "smb" entry to somewhere else in the add-ons array so it can be sorted in alphabetical order if possible
- - For future reference, for pull requests, please please the name of the add-on and its version in the pull request title
-   This is my mistake, and I'll correct now.

**Comment by josephsl** [comment]
Hi, okay, this gets interesting. I don’t know about others, but I think it would be much better if SMB comes up with a way to make their program accessible without resorting to writing NVDA add-ons to make them usable. I think, for this case, writing add-ons does not really address the “real” issue at hand as the deeper problem may have to do with the way the program is presenting the user interface (perhaps might be using a GUI toolkit known not to be screen reader user friendly). At least writing and submitting add-ons may provide a way to learn about add-on creation, but it does not remove the responsibility of the program developers themselves to improve things not only for NVDA users, but also for users of other screen readers. Thanks.

File: get.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 $addons = array(
-	"access8math" => "https://github.com/tsengwoody/Access8Math/releases/download/v3.3/Access8Math-3.3.nvda-addon",
+		"access8math" => "https://github.com/tsengwoody/Access8Math/releases/download/v3.3/Access8Math-3.3.nvda-addon",
```
Comment (seanbudd) [inline get.php:3]: It looks like this line was mistakenly changed


```suggestion
	"access8math" => "https://github.com/tsengwoody/Access8Math/releases/download/v3.3/Access8Math-3.3.nvda-addon",
```

**Review by seanbudd — APPROVED** [review]

**Pull request merged**